dds_cli/__main__.py

```diff
@@ -372,356 +264,243 @@ def ls(click_ctx, project, folder, projects, size, username, usage, sort, tree,
-                                file_downloaded = dfut.result()
-                                LOG.debug(
-                                    f"Download of {downloaded_file} successful: {file_downloaded}"
-                                )
-                            except concurrent.futures.BrokenExecutor as err:
-                                LOG.critical(
-                                    f"Download of file {downloaded_file} failed! Error: {err}"
-                                )
-                                continue
-
-                            new_tasks += 1
-                            progress.advance(task_dwnld)
-
-                        # Schedule the next set of futures for download
-                        for next_file in itertools.islice(iterator, new_tasks):
-                            LOG.debug(f"Starting: {next_file}")
-                            # Execute download
-                            download_threads[
-                                texec.submit(
-                                    getter.download_and_verify,
-                                    file=next_file,
-                                    progress=progress,
-                                )
-                            ] = next_file
-    except (
-        dds_cli.exceptions.InvalidMethodError,
-        OSError,
-        dds_cli.exceptions.TokenNotFoundError,
-        dds_cli.exceptions.AuthenticationError,
-        dds_cli.exceptions.ApiRequestError,
-        dds_cli.exceptions.ApiResponseError,
-        SystemExit,
-        dds_cli.exceptions.DDSCLIException,
-        dds_cli.exceptions.NoDataError,
-        dds_cli.exceptions.DownloadError,
-    ) as err:
-        LOG.error(err)
-        sys.exit(1)
+        except (
+            dds_cli.exceptions.AuthenticationError,
+            dds_cli.exceptions.ApiResponseError,
+            dds_cli.exceptions.ApiRequestError,
+            dds_cli.exceptions.DDSCLIException,
+        ) as err:
+            LOG.error(err)
+            sys.exit(1)
 
 
-# COMMAND##########################################################################
-#### AUTH ################################################################## AUTH #
-###################################################################################
-@dds_main.group()
-@click.pass_obj
-def auth(_):
-    """Manage the saved authentication token."""
+####################################################################################################
+####################################################################################################
+## PROJECT ############################################################################## PROJECT ##
+####################################################################################################
+####################################################################################################
 
 
-@auth.command()
-@common_options
+@dds_main.group(name="project", invoke_without_command=True)
```

Comment:
Maybe `no_args_is_help=True` could be added here too.

Reply:
Added that. But removed the `invoke_without_command`. Unnecessary.